File: iniciante/javascript/1019.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let entrada = parseInt(lines.shift())
let horas = parseInt(entrada / 3600)
entrada %= 3600
let minutos = parseInt(entrada / 60)
entrada %= 60
let segundos = entrada
console.log(`${horas}:${minutos}:${segundos}`)

File: iniciante/python/1019.py
# -*- coding: utf-8 -*-

entrada = int(input())

horas = 0
minutos = 0
segundos = 0
contador = 0

while contador < entrada:
    contador += 1
    if contador == 60:
        minutos += 1
        entrada -= contador
        contador = 0
    if minutos == 60:
        horas += 1
        minutos = 0

segundos = contador

print(f'{horas}:{minutos}:{segundos}')
